open-company/open-company-api
# routing changes

- trello card: https://trello.com/c/OfCMnDsC/343-routing-changes

### To Test:

- Run tests
- Try creating a company called `about` or `api` or any of the other reserved slugs, notice that the resulting slug gets modified (a  number is appended)
- look over the other modifications which allow customizing `/companies` for logged in users

### Discussion

1. Should we keep the code to customize `/companies`? Since all the redirect-on-login stuff only depends on what the entry point returns those changes are not strictly required. I think they might be useful in the future though. We can keep the new code and also keep the old behavior if that's wanted.
